<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdaterOptionsPage</class>
 <widget class="QWidget" name="AutoUpdaterOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Auto Updater Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Auto Updating Interval (0 for disabled):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="interval">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Update to pre releases:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="usePreReleases">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>GitHub API URL (advanced use):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="gitHubAPIUrl"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="gitHubUsername"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="gitHubPassword">
        <property name="toolTip">
         <string>Warning! Password is stored in cleartext!</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>usePreReleases</sender>
   <signal>toggled(bool)</signal>
   <receiver>gitHubPassword</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePreReleases</sender>
   <signal>toggled(bool)</signal>
   <receiver>gitHubUsername</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePreReleases</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePreReleases</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
